antora-playbook.yml

```diff
@@ -139,6 +139,8 @@ asciidoc:
     xrefstyle: short
     enterprise: https://www.couchbase.com/products/editions[ENTERPRISE EDITION]
     community: https://www.couchbase.com/products/editions[COMMUNITY EDITION]
+    kroki-server-url: http://3.91.133.254:9500/
```

Comment:
I tried to access this from my local (VPN on) and couldn't see anything, is that url correct?

Reply:
Hi 👋🏾. 

@maria-robobug  <-- Yes the URL is correct, but you can't see it unless you're signed into `vpn-public.couchbase.com`

If you add `vpn-public.couchbase.com` to your GlobalProtect setup and sign in on that, then the URL will show up

Reply:
ah ok, I only have: `vpn-uk.couchbase.com` and `vpn.couchbase.com`, I'll add 👍 .

Reply:
that works, nice one!

Reply:
@maria-robobug <-- I'll wait until I know the Jenkins server can call the Kroki instance before asking you to merge